<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DEPORTS_SANS_VARIATION_ENTRAXE</class>
 <widget class="QWidget" name="DEPORTS_SANS_VARIATION_ENTRAXE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DEPORTS SANS VARIATION D'ENTRAXE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>reducteur_gearexpert.gif</normaloff>reducteur_gearexpert.gif</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton_3_SANS_VARIATION">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>METHODE DE MAAG-TASCHENBUCH</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton_SANS_VARIATION">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>METHODE ISO</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton_2_SANS_VARIATION">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>METHODE DE W.RICHTER</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton_4_SANS_VARIATION">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>RETOUR</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
